modules/javafx.graphics/src/main/java/javafx/scene/text/TextFlow.java

```diff
@@ -193,11 +193,15 @@ private void checkOrientation() {
 
     /**
      * Maps local point to {@link HitInfo} in the content.
+     * <p>
+     * NOTE: this method may return incorrect value with non-empty border or padding.
```

Comment:
You should be more explicit, and say that it will not take border or padding into account.